source/support/communicating-with-users/common-responses.html.md

```diff
@@ -459,10 +459,11 @@ This will send you emails from every team that uses this platform so if you woul
 
 Teams from other organisations occasionally ask us to speak to them about the GOV.UK Design System. For example, teams in central and local government, the third sector and sometimes even the private sector.
 
-When we receive a meeting request, use our [meeting requests form] to help:
+When we receive a meeting request, ask the team on Slack whether there's interest in talking to the organisation. If so, delegate the response to those person(s).
```

Comment:
Minor housekeeping thing, as the 'meeting requests form' link has been removed in this change, you can remove its definition on line 584 of this file.

Reply:
Good spot!